scripts/serve/README.md

```diff
@@ -0,0 +1,83 @@
+# Terra Toolkit Serve
+
+Terra Toolkit offers up two different commands to serve your client side application, `serve` and `serve-static`.
+If you are using terra-dev-site, use the provided `tds:serve`/`tds:serve-static` commands instead.
+
+## serve
+Serve is a replacment for webpack-dev-server. Behind the scenes it's using [webpack-serve](https://github.com/webpack-contrib/webpack-serve).
+Serve is a hot reloading server and does not work on IE 10 or below. See [compatible browsers](https://caniuse.com/#feat=websockets). Use serve-static for IE 10 testing.
+
+Serve is offered up as both a cli and a javascript function.
+
+### CLI
+#### API
+| Name  | Required | Default Value | Description |
+| ------------- | ------------- | ------------- | ------------- |
+| **--config**  | false | `undefined` | The webpack config to serve. |
+| **--port**  | false | `8080` | The port the server should listen on. |
+| **-p, --production | false | false | Passes the -p flag to the webpack config. |
+
+If no config is supplied tt:serve will first search for `webpack.config.js` in the working directory, if that is not found it will attempt to use the default webpack config supplied by terra-dev-site.
+
+#### In your package.json
+```JSON
+{
+  "start": "tt:serve --config src/webpack/webpack.config"
+}
+```
+### Function
+#### API
+| Name  | Required | Default Value | Description |
+| ------------- | ------------- | ------------- | ------------- |
```

Comment:
should index be added as an option?

Reply:
I'd rather not until someone needs it. I only kept it to stay passive with express dev server service.